wallet/src/main/java/com/android/identity_credential/wallet/MainActivity.kt

```diff
@@ -265,15 +276,35 @@ class MainActivity : ComponentActivity() {
             if (application.credentialStore.listCredentials().size == 0) {
                 MainScreenNoCredentialsAvailable(navigation)
             } else {
-                MainScreenCredentialPager(navigation)
-                Row(
-                    modifier = Modifier.fillMaxWidth(),
-                    horizontalArrangement = Arrangement.Center
+                Box(
+                    modifier = Modifier.fillMaxSize()
                 ) {
-                    Text(
-                        modifier = Modifier.padding(8.dp),
-                        text = "Hold to Reader"
-                    )
+                    Column(modifier = Modifier.align(Alignment.Center))
+                    {
+                        MainScreenCredentialPager(navigation)
+                        Row(
+                            modifier = Modifier.fillMaxWidth(),
+                            horizontalArrangement = Arrangement.Center
+                        ) {
+                            Text(
+                                modifier = Modifier.padding(8.dp),
+                                text = "Hold to Reader"
+                            )
+                        }
+                    }
+                    Button(
+                        onClick = {
+                            qrEngagementViewModel.generateQrEngagementHelper (
+                                onComplete = { navigation.navigate("QrEngagementScreen") },
+                                onConnected = { navigation.popBackStack() })
+                        },
+                        modifier = Modifier.align(Alignment.BottomCenter)
+                    ) {
+                        Text(
+                            modifier = Modifier.padding(8.dp),
+                            text = "Show QR"
+                        )
```

Comment:
It's kinda jarring how there's no padding here, please add some.

![image](https://github.com/openwallet-foundation-labs/identity-credential/assets/5580198/57e6334a-eb3b-4d5c-a6c5-f2bac08102fd)